addons/website/views/snippets/s_key_benefits.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_key_benefits" name="Key benefits">
+    <section class="s_key_benefits pt48 pb48">
+        <div class="container">
+            <div class="row o_grid_mode s_nb_column_fixed" data-row-count="11" style="column-gap: 48px;">
+                <div class="o_grid_item g-col-lg-4 g-height-1 col-lg-4" style="--grid-item-padding-x: 16px; --grid-item-padding-y: 0px; grid-area: 1 / 1 / 2 / 5; z-index: 1;">
+                    <p class="lead">
+                        <span>✽</span>&#160;&#160;What We Offer
```

Comment:
Just to be sure, the `<span>` is there to isolate the icon from the text in the translation ?
Or to make it possible to `<xpath>` the icon specifically in themes ?

By the way, isn't `✽` too "flowery" for the most sober themes ?

Reply:
As this was kind of "special character", I added a span to isolate it from the "regular text", but if it does not make sense as this is not how the editor would render it by default, we can remove it.

About the character being too flowery, I think it's fine as this is part of the default theme and this is probably the first thing will change.

Reply:
Given that even when using the `/emoji` command there is no wrapping, I guess the `span` is not needed indeed.

Note that there are conflicts on the design themes branch, a rebase is needed.